plugins/modules/source_control/gitlab/gitlab_service.py

```diff
@@ -0,0 +1,468 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Setup or delete GitLab integration services
+version_added: '2.4.0'
+description:
+  - Creates, updates, or deletes GitLab integrations formerly known as "services".
+author:
+  - Raphaël Droz (@drzraf)
+requirements:
+  - python-gitlab python module
+extends_documentation_fragment:
+- community.general.auth_basic
+
+options:
+  api_token:
+    description:
+      - GitLab token for logging in.
+    type: str
+  project:
+    description:
+      - The ID or path of the project (urlencoded or not).
+    required: true
+    type: str
+  active:
+    description:
+      - Not yet supported (U(https://gitlab.com/gitlab-org/gitlab-ce/issues/41113)).
```

Comment:
Why add this option already now if it is not supported? What happens if a user decides to use this option?

Reply:
- Over-anticipation?
- It's easier to have it now (and just uncomment once available at GitLab) than some other developer trying to get it from scratch again later.

> What happens if a user decides to use this option?
If someone set active to False, the service will still be setup **and** "activated" at GitLab anyway.

The benefit of the presence of this parameter is that sysadmins can keep track/document their infrastructure and the settings they want (like `This service must NOT be enabled because of ticket #....`), but the Ansible plugin will not enforce it.

Reply:
I don't think this is something we want to start doing. I could understand if this option comes from a docs fragment / module_utils and cannot be implemented for all modules at once, but here it is only for this module. If you want to leave that part here, please comment it out.